src/EnergyPlus/DataAirflowNetwork.cc

```diff
@@ -186,6 +191,9 @@ namespace DataAirflowNetwork {
 	// Addiitonal airflow needed for an VAV fan to compensate the leakage losses and supply pathway pressure losses [kg/s]
 	Real64 VAVTerminalRatio( 0.0 ); // The terminal flow ratio when a supply VAV fan reach its max flow rate
 	bool VAVSystem( false ); // This flag is used to represent a VAV system
+	Real64 ExhaustFanMassFlowRate( 0.0 ); // Exhaust fan flow rate used in PressureStat
+	int PressureSetFlag( 0 ); // PressureSet flag
```

Comment:
Not clear what this flag is for.  Does this mean there is pressure control in the input, or that the pressure has already been set, or ?